website/server/controllers/api-v3/user.js

```diff
@@ -137,6 +142,16 @@ let checkPreferencePurchase = (user, path, item) => {
   return _.get(user.purchased, itemPath);
 };
 
+function _setNewStuffFlag (val, res) {
+  if (val === true) {
+    return moment(res.locals.habitrpg.NEWS_DATE).subtract(1, 'days').toDate();
```

Comment:
Again, probably best to use nconf here. Maybe we can move this function to the user model?